src/cascade/model/dismod_groups.py

```diff
@@ -32,7 +32,7 @@ def __setattr__(self, item, value):
         if item in self.GROUPS and self.__dict__.get("_frozen", False):
             raise AttributeError(f"Cannot set attribute")
         else:
-            self.__dict__[item] = value
+            object.__setattr__(self, item, value)
```

Comment:
This was a real bug. When a setattr hides members, if it assigns to the __dict__ directly, then @property values on the children won't work correctly. Using the object setattr makes the @property for scale work on the child.